src/exporter.js

```diff
@@ -52,7 +44,7 @@ function exporter (hash, dagService, options, callback) {
         rs.push(unmarshaledData.data)
         rs.push(null)
       }
-      ee.emit('file', { stream: rs, path: name, dir: dir })
+      this.push({ stream: rs, path: name, dir: dir })
```

Comment:
- `path` is the full path to the file
- `stream` is the readable stream
- why is dir pushed here ?